Bottom: 7c77f7ca0658586067dae0188894df8aad8338fe
Top:    2e9f92c34964db7ee89774a329bec556c0b77fc2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-03-21 11:50:57 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
